Add deno runtime (#2)
apache/openwhisk-runtime-deno: README.md

<!--
#
# Licensed to the Apache Software Foundation (ASF) under one or more
# contributor license agreements.  See the NOTICE file distributed with
# this work for additional information regarding copyright ownership.
# The ASF licenses this file to You under the Apache License, Version 2.0
# (the "License"); you may not use this file except in compliance with
# the License.  You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#
-->

# OpenWhisk Runtime for Deno

> Try it out using docker image:

Reviewer: ```suggestion
Try it out using a Docker image. Create a file `main.ts`:
```
lift main.ts here and follow it with wsk command.

```sh

> wsk action create hello-deno main.ts --docker=satyarohith/openwhisk-runtime-deno

Reviewer: ```suggestion
wsk action create hello main.ts --docker=openwhisk/action-deno-1.3.0
```

```

> The content of the `main.ts` is as bellow.

Reviewer: ```suggestion
The content of the `main.ts` is shown below.
```

Reviewer: but see note above.
